content/en/docs/getting-started/_index.html

```diff
@@ -0,0 +1,246 @@
+        <a class="nav-link" data-toggle="tab" href="#linux">Linux</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" data-toggle="tab" href="#windows">Windows</a>
+      </li>
+    </ul>
+    <div class="tab-content" id="install-jx-tabs">
+      <div class="tab-pane fade show active" id="osx">
+        <h3>Using Homebrew</h3>
+
+{{<highlight sh>}}
+brew install jenkins-x/jx/jx
+{{</highlight>}}
+
+        <h3>Using Curl</h3>
+
+{{<highlight sh>}}
+# Download the `.tar` file, unarchive it.
+curl -L "https://github.com/jenkins-x/jx/releases/download/$(curl --silent https://api.github.com/repos/jenkins-x/jx/releases/latest | jq -r '.tag_name')/jx-darwin-amd64.tar.gz" | tar xzv "jx"
+
+# Add `jx` to your path
+sudo mv jx /usr/local/bin
+
+# verify installation
+jx --version
+{{</highlight>}}
+
+      </div>
+      <div class="tab-pane fade" id="linux">
+
+        &nbsp;
+        
+{{<highlight sh>}}
+# Download the `.tar` file, and unarchive it in a directory where you can run the `jx` command.
+curl -L "https://github.com/jenkins-x/jx/releases/download/$(curl --silent "https://github.com/jenkins-x/jx/releases/latest" | sed 's#.*tag/\(.*\)\".*#\1#')/jx-linux-amd64.tar.gz" | tar xzv "jx"
+
+# Move binary to your path
+sudo mv jx /usr/local/bin
+
+# verify installation
+jx --version
+{{</highlight>}}
+
+      </div>
+      <div class="tab-pane fade" id="windows">
+        &nbsp;
+
+        <h3> Using Chocolatey</h3>
+
+        1. Right-click menu:Start\[Command Prompt (Admin)\].
+        <br><br>
+        2. At the shell prompt, execute a `powershell.exe` command to download
+        and install the `choco` binary and set the installation path so that
+        the binary can be executed:
+
+{{<highlight sh>}}
+@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile `
+-InputFormat None -ExecutionPolicy Bypass `
+-Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" `
+&& SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+{{</highlight>}}
+
+        3. Install Jenkins X using Chocolatey:
+
+{{<highlight sh>}}
+choco install jenkins-x
+{{</highlight>}}
+
+      </div>
+    </div>
+  </div>
+</div>
+
+<p>&nbsp;</p>
+<p>&nbsp;</p>
+
+<div class="row">
+  <div class="col col-lg-12">
+    <p>&nbsp;</p>
+    <h2><strong>Step 3: Install Jenkins X on your cluster</strong></h2>
+
+    <p>
+      You have provisioned your Kubernetes cluster and installed the <code>jx</code> CLI.
+      Now you are ready to install Jenkins X into the cluster.
+      To do that, you will use the <code>jx boot</code> command.
+      For more details around JX Boot refer to the <a href="/docs/getting-started/setup/boot">Run Boot</a> section.
+      In an empty directory run:
+    </p>
+
+{{<highlight bash>}}
+# the parameter here, is the requirements yaml file
+# which was saved in the same directory you executed terraform apply.
+jx boot -r <path-to-terraform-directory>/jx-requirements.yml
```

Comment:
Please don't couple Terraform and JX Boot. One is not the requirement for the other, at least not until Terraform supports all the platforms where JX can run (which will probably never happen). I think that references like `path-to-terraform-directory` should be removed (there can be a note that's the option), and that this page should be split. Terraform is a helper to create a cluster, not a requirements. People should be able to install JX no matter how they created a cluster.

Reply:
Agreed with Viktor

Reply:
This is just in the context of this getting started guide where there are three successive steps to get started using the Terraform modules.

I agree that Terraform is not a requirement per se and it should be made clear. I am also inclined to think now that generating the initial `jx-requirements.yml` is taking it a bit too far. The disadvantages might actually outweigh the benefits. See also https://github.com/jenkins-x/terraform-google-jx/issues/75